File: Sources/gestion-culturelle-front/project/src/app/model/adminDto.ts
import { RoleDto } from './roleDto';

export class AdminDto {
    id: number;
    nom: string;
    prenom: string;
    email: string;  
    password: string;
    adresse: string;
    role: RoleDto;

    constructor(
        id?: number,
        nom?: string,
        prenom?: string,
        email?: string,       
        password?: string,
        adresse?: string,
        role?: RoleDto,
    ) {
        this.id = id;
        this.nom = nom;       
        this.email = email;    
        this.prenom = prenom;
        this.password = password;
        this.adresse = adresse;
        this.role = role;


    }
}

File: Sources/gestion-culturelle-back/src/main/java/com/afpa/cda/dto/AdminDto.java
package com.afpa.cda.dto;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;
import lombok.ToString;

@NoArgsConstructor
@AllArgsConstructor
@Getter
@Setter
@Builder
@ToString
public class AdminDto {
	private int id;
	private String nom;
	private String prenom;
	private String email;
	private String password;
	private String tokenSecret;
	private String adresse;
	private RoleDto role;
	private String photo;
}
